nginx.template

```diff
@@ -18,6 +18,14 @@ server {
 
   keepalive_timeout   70;
   add_header          Alternate-Protocol  $NGINX_SSL_PORT:npn-spdy/2;
+
+  location ~* ^.+.(jpg|jpeg|png|gif|ico|svg|css|js)$ {
+    expires 30d;
```

Comment:
What is this? 30d cache for js files?

Reply:
we don't use nginx anymore for our servers, so this file is not relevant

Reply:
Will remove it